platform/ruby/mail_checker.rb

```diff
@@ -0,0 +1,35 @@
il.com","r4nd0m.de","raetp9.com","raketenmann.de","rancidhome.net","raqid.com","rax.la","raxtest.com","recipeforfailure.com","reconmail.com","redfeathercrow.com","remarkable.rocks","remote.li","reptilegenetics.com","revolvingdoorhoax.org","riddermark.de","risingsuntouch.com","rnailinator.com","robertspcrepair.com","ronnierage.net","rotaniliam.com","rowe-solutions.com","royaldoodles.org","rumgel.com","rustydoor.com","s33db0x.com","sabrestlouis.com","sackboii.com","saharanightstempe.com","samsclass.info","sandwhichvideo.com","sanfinder.com","sanim.net ","sanstr.com","satukosong.com","sausen.com","schachrol.com","sd3.in","secured-link.net","seekapps.com","sejaa.lv","sendfree.org","sendingspecialflyers.com","sexforswingers.com","sexical.com","shhmail.com","shhuut.org","shipfromto.com","shiphazmat.org","shipping-regulations.com","shippingterms.org","shrib.com","simpleitsecurity.info","sinfiltro.cl","singlespride.com","sizzlemctwizzle.com","skkk.edu.my","sky-inbox.com","slothmail.net","smtp99.com","smwg.info","socialfurry.org","soisz.com","solventtrap.wiki","spam.org.es","spamlot.net","speedgaus.net","spritzzone.de","stanfordujjain.com","starlight-breaker.net","startfu.com","statdvr.com","stathost.net","statiix.com","steambot.net","streetwisemail.com","stumpfwerk.com","suburbanthug.com","suckmyd.com","sylvannet.com","tafoi.gr","tagmymedia.com","tanukis.org","tb-on-line.net","telecomix.pl","testudine.com","theaviors.com","thebearshark.com","thediamants.org","thembones.com.au","themostemail.com","thescrappermovie.com","theteastory.info","thietbivanphong.asia","thisurl.website","thnikka.com","thunkinator.org","thxmate.com","timgiarevn.com","timkassouf.com","tinyurl24.com","tlpn.org","tmpjr.me","toddsbighug.com","tokenmail.de","tonymanso.com","top101.de","topofertasdehoy.com","toprumours.com","toss.pw","totalvista.com","totesmail.com","tp-qa-mail.com","tranceversal.com","trasz.com","trollproject.com","tropicalbass.info","trungtamtoeic.com","ttszuo.xyz","tualias.com","txtadvertise.com","ufacturing.com","uguuchantele.com","uhhu.ru","unimark.org","unit7lahaina.com","uploadnolimit.com","urfunktion.se","utiket.us","uwork4.us","vaati.org","valhalladev.com","verdejo.com","veryday.ch","veryday.eu","veryday.info","victoriantwins.com","vikingsonly.com","vinernet.com","vipxm.net ","vixletdev.com","vmailing.info","vmani.com ","vmpanda.com","vorga.org","votiputox.org","voxelcore.com","wakingupesther.com","watchever.biz","watchironman3onlinefreefullmovie.com","wbml.net","webtrip.ch","welikecookies.com","wg0.com","whatifanalytics.com","whiffles.org","wibblesmith.com","widget.gg","wimsg.com ","wralawfirm.com","x1x.spb.ru","x24.com","xcompress.com","xcpy.com","xjoi.com","xn--9kq967o.com","xrho.com","xwaretech.com","xwaretech.info","xwaretech.net","xww.ro","yaqp.com","ynmrealty.com","yougotgoated.com","youneedmore.info","yourewronghereswhy.com","yourlms.biz","yspend.com","yugasandrika.com","yui.it","zepp.dk","zipsendtest.com","zoetropes.org","zombie-hive.com","zumpul.com"].
```

Comment:
If every platform has it's own list that will be hard to maintain and keep it consolidated. Maybe JS, Node, PHP, Ruby, and others should all read from the `list.json` available on the root of the repository?